src/components/Account/index.js

```diff
@@ -8,81 +8,153 @@ import {
 } from 'react-bootstrap';
 
-const Account = () => (
-  <div className={'Account'}>
-    <div className="container">
-      <div className="accountCard card">
-        <h1>
-          Account Settings
-        </h1>
+const Account = ({ account, updateField, updateAccount, isUpdate }) => {
+  const resumeSiteUrl = account.resume_site_url;
+  const linkedinUrl = account.linked_in_url;
+  const stackoverflowUrl = account.stackoverflow_url;
+  const twitterHandle = account.twitter_handle;
+  const githubHandle = account.github_handle;
+  const codewarsHandle = account.codewars_handle;
 
-        <Form horizontal>
-          <FormGroup controlId="formHorizontalResumeSiteUrl">
-            <Col componentClass={ControlLabel} sm={10} md={6}>
-              Resume Site Url
-            </Col>
-            <Col sm={10} md={6}>
-              <FormControl type="text" placeholder="http://..." />
-            </Col>
-          </FormGroup>
+  const handleChange = (field, event) => {
+    updateField(field, event.target.value);
+  };
 
-          <FormGroup controlId="formHorizontalLinkedInUrl">
-            <Col componentClass={ControlLabel} sm={10} md={6}>
-              LinkedIn Profile Url
-            </Col>
-            <Col sm={10} md={6}>
-              <FormControl type="text" placeholder="https://linkedin.com/in/..." />
-            </Col>
-          </FormGroup>
+  const handleSubmit = () => {
+    const params = {
+      resume_site_url: resumeSiteUrl,
+      linkedin_url: linkedinUrl,
+      stackoverflow_url: stackoverflowUrl,
+      twitter_handle: twitterHandle,
+      github_handle: githubHandle,
+      codewars_handle: codewarsHandle,
+    };
 
-          <FormGroup controlId="formHorizontalStackOverflowUrl">
-            <Col componentClass={ControlLabel} sm={10} md={6}>
-              Stack Overflow Profile Url
-            </Col>
-            <Col sm={10} md={6}>
-              <FormControl type="text" placeholder="https://stackoverflow.com/users/..." />
-            </Col>
-          </FormGroup>
+    updateAccount(params);
+  };
 
-          <FormGroup controlId="formHorizontalTwitterHandle">
-            <Col componentClass={ControlLabel} sm={10} md={6}>
-              Twitter Handle
-            </Col>
-            <Col sm={10} md={6}>
-              <FormControl type="text" placeholder="your twitter @handle" />
-            </Col>
-          </FormGroup>
+  return (
+    <div className={'Account'}>
+      <div className="container">
+        <div className="accountCard card">
+          <h1>
+            Account Settings
+          </h1>
 
-          <FormGroup controlId="formHorizontalGithubHandle">
-            <Col componentClass={ControlLabel} sm={10} md={6}>
-              GitHub Handle
-            </Col>
-            <Col sm={10} md={6}>
-              <FormControl type="text" placeholder="your github @handle" />
-            </Col>
-          </FormGroup>
+          {
+            isUpdate && <span>Updated successfully</span>
```

Comment:
FYI, this pops up when a successful account update occurs.  Up to you, @hamholla if you want to add styling.